llvm/utils/release/test-release.sh

```diff
@@ -528,11 +528,11 @@ function build_llvmCore() {
     LitVerbose="-v"
 
     InstallTarget="install"
-    if [ "$Phase" -lt "3" ]; then
+    # compiler-rt builtins is needed on AIX to have a functional Phase 1 clang.
+    if [ "$Phase" -lt "3" ] && [ "$System" != "AIX" ]; then
```

Comment:
This is insulating AIX from the broader effects of the code here (to reduce the phase 1 and 2 builds) entirely. It should be possible to just add the necessary additional targets for AIX.

Reply:
Yes, this is more of a temporary solution for now since I'm still not sure why the original code is skipping the compiler-rt build during phase 1. Once I've investigated the problem more I'll update the fix.